live-examples/css-examples/grid/grid-column-start.html

```diff
@@ -0,0 +1,32 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="grid-column-start">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">grid-column-start: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">grid-column-start: 2;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">grid-column-start: -1;</code></pre>
```

Comment:
Would it be useful here to also have an example of `grid-column-start: span 3;` as you did for `grid-column-end`?